tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -354,6 +361,8 @@ def emit_view_lambda(f: NativeFunction, unpacked_bindings: List[Binding]) -> str
                 arg=arg, val=arg_value, default="0"
             )
             updated_unpacked_args.append(arg_value)
+        elif arg_type == ConstRefCType(BaseCType(tensorT)):
+            updated_unpacked_args.append(arg[:-1])
```

Comment:
I hate this and this is hacky. Seems like we will be eventually getting to sym ints, but in the interim we are doing tensors. I started on working on adding the `unpacking` of all tensor args to the ad_inplace_or_view body but would love a double check that this is what I should do. Not sure where `unpack` is defined so that I can include it in pytorch/torch/csrc/autograd/generated/ADInplaceOrViewTypeEverything.cpp

Reply:
idk I did the unpack thing and it is also a pain and not pain and feels like a pain. I will wait till someone confirms me on the unpack route